hudi-common/src/main/java/org/apache/hudi/common/model/HoodieCommitMetadata.java

```diff
@@ -45,6 +45,13 @@
 
 /**
  * All the metadata that gets stored along with a commit.
+ * ******** IMPORTANT ********
+ * For any newly added/removed data fields, make sure we have the same definition in
+ * src/main/avro/HoodieCommitMetadata.avsc file!!!!!
+ *
+ * For any newly added subclass, make sure we add corresponding handler in
+ * org.apache.hudi.common.table.timeline.versioning.v2.CommitMetadataSerDeV2#deserialize method.
+ * ***************************
```

Comment:
I assume this class is still need to commit metadata SerDe V1 which does not leverage the Avro schema, correct?

Reply:
If we could use Avro schema for JSON SerDe as well, it's better to remove this class completely.